images/win/Windows2022-Readme.md

```diff
@@ -69,7 +69,7 @@
 - Kind 0.14.0
 - Kubectl 1.24.1
 - Mercurial 5.0
-- Mingw-w64 8.1.0
+- Mingw-w64 11.2.0-07112021 
```

Comment:
@abrown no need to update the readme. It is autogenerated

Reply:
Ok, I reverted that commit. Can you approve the CI run?